weblate/trans/models/component.py

```diff
@@ -1176,6 +1207,118 @@ def get_repoweb_link(
             component=self,
         )
 
+    def get_git_repoweb_template(self):
+        if any(regex.match(self.repo) for regex in BITBUCKET_GIT_REPOS_REGEXP):
+            return self.get_bitbucket_git_repoweb_template()
+
+        if any(regex.match(self.repo) for regex in BITBUCKET_HG_REPOS_REGEXP):
+            raise self.get_bitbucket_hg_repoweb_template
```

Comment:
BitBucket no longer supports Mercurial (see https://bitbucket.org/blog/sunsetting-mercurial-support-in-bitbucket), so this is not needed.

Reply:
okay, removed